Source/WebCore/html/HTMLScriptElement.idl

```diff
@@ -20,14 +20,16 @@
 [
     Exposed=Window
 ] interface HTMLScriptElement : HTMLElement {
-    [CEReactions=NotNeeded] attribute ScriptString text;
+    [CEReactions=NotNeeded] attribute (TrustedScript or DOMString) text;
+    [CEReactions=NotNeeded, ImplementedAs=innerTextForBindings] attribute (TrustedScript or [LegacyNullToEmptyString] DOMString) innerText;
```

Comment:
I don’t think we need `ImplementedAs`, we can just overload `setInnerText`. That will save us from having to write `innerTextForBindings` at all.